docs/source/main_classes/trainer.mdx

```diff
@@ -189,6 +189,98 @@ that make things deterministic (.e.g., `torch.backends.cudnn.deterministic`) may
 can't be done by default, but you can enable those yourself if needed.
 
 
+## Specific GPUs Selection
+
+Let's discuss how you can tell your program which GPUs are to be used and in what order.
+
+When using `DistributedDataParallel` to use only a subset of your GPUs, you simply specify the number of GPUs to use. For example, if you have 4 GPUs, but you wish to use the first 2 you can do:
+
+```bash
+python -m torch.distributed.launch --nproc_per_node=2  trainer-program.py ...
+```
+
+You can do the same with the `deepspeed` launcher if you have it installed:
+
+```bash
+deepspeed --num_gpus 2 trainer-program.py ...
+```
+You don't need to use [the Deepspeed integration](deepspeed) to use the `deepspeed` launcher:
+
```

Comment:
We could mention the Accelerate launcher here as well:
```
accelerate launch trainer-program.py
```
(Same as DeepSpeed, you don't need to be using Accelerate to use the launcher)

Reply:
but then you probably want to show how to activate more than one gpu? i.e. `--num_gpus 2`

I tried to find how to do that but I don't see anything here:
https://huggingface.co/docs/accelerate/quicktour.html#launching-your-distributed-script

Reply:
It controls that though the config of the user, or they can ask for less with `--num_processes 2` if they want.

Reply:
done.

Reply:
Thank you!